agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1067,10 +1069,12 @@ bool main_thread::backhaul_fsm_wireless(bool &skip_select)
                 success = false;
                 break;
             }
+            scan_trigerred = true;
```

Comment:
I think that if you will define the 'bool success = false` and here you will set it to true, you will get the same result.

Reply:
Thanks for your comment.

I want to differentiate between failure to perform the scan and unavailability of STAs to perform the scan (although both result in BH manager restart